Dockerfile

```diff
@@ -1,24 +1,23 @@
 FROM node:16-buster as build
 WORKDIR /usr/src/monorepo
-COPY . .
 RUN npm set unsafe-perm true && \
-	# explicitly use npm v6
-	npm install -g npm@6 && \
-	npm ci && \
-	npm run bootstrap-pkg streamr-broker
+	# explicitly use npm v8
+	npm install -g npm@8 --no-audit --progress=false
+COPY ["./*.json", "./*.js", "./*.mjs", ".npmrc",  ".gitignore", "./"]
+RUN npm ci
+COPY ["./packages", "./packages"]
+RUN npm run bootstrap-pkg -- streamr-broker
 
 # image contains all packages, remove devDeps to keep image size down
 # --ignore-scripts as sqlite package in the client tries running its
 # 'install' script, which uses node-pre-gyp, which is a devDependency that
 # gets removed by prune.
-RUN npx lerna exec -- npm prune --production --ignore-scripts && \
+RUN npx lerna exec --parallel --include-dependencies --scope "streamr-broker" -- npm prune --production --ignore-scripts --prefer-offline --no-audit && \
 	# restore inter-package symlinks removed by npm prune
 	npx lerna link
 
-FROM node:16-buster-slim
-RUN apt-get update && apt-get install --assume-yes --no-install-recommends curl \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
+FROM node:16-alpine
```

Comment:
Please use Debian based images for Docker containers. Currently `buster` or `buster-slim`.

Reasoning is to minimise download times when pulling containers from Hub. This is achieved by sharing Docker container layers. Currently we aim to share the base layer and maybe later we can share more.

Following command creates a single layer that can be shared by other containers:
```
RUN apt-get install -y curl
```

Reply:
Ah good point, though the reason I switched to alpine is the image is only ~5MB vs ~25MB for buster-slim.

Reply:
Given that all Docker containers use Debian Buster base image I don't think the size of the base image is significant.

Reply:
Changed.